onboarding/wordpress.md

```diff
@@ -0,0 +1,46 @@
+We recommend installing the Rollbar plugin through <a href="https://wpackagist.org/" target="_blank" rel="noopener">wpackagist</a> if you manage your project using Composer. This way ensures the plugin and all of its' dependencies are managed by Composer.
+
+
+1. If your WordPress project is not managed with Composer yet, we suggest looking into upgrading your WordPress: <a href="https://roots.io/using-composer-with-wordpress/" target="_blank" rel="noopener">Using Composer with WordPress</a>.
+
+2. In your `composer.json` add `wpackagist-plugin/rollbar` to your `require` section, i.e.:
+ 
+```
+ "require": {
+ "php": ">=5.5",
+ ...,
+ "wpackagist-plugin/rollbar": "*"
+  }
+```
+
+3. Issue command `composer install` in the root directory of your WordPress project.
+
+4. In `Plugins → Installed plugins` find `Rollbar` and click `Activate` underneath.
```

Comment:
That utf8 character miiiight cause us problems - does this work ok in your dev environment? You might want/need to use the html entity version instead